angles/src/angles/__init__.py

```diff
@@ -34,18 +34,10 @@
 
 from math import fmod, pi, fabs
 
-def normalize_angle_positive(angle):
```

Comment:
We can't remove this public function. This is failing the CI tests due to us having coverage of this deleted function.